onnx/defs/tensor/utils.cc

```diff
@@ -236,6 +240,14 @@ void resizeShapeInference(InferenceContext& ctx) {
   } // nullptr != scales
 }
 
+void resizeShapeInference_opset13_to_18(InferenceContext& ctx) {
+  resizeShapeInferenceVersioned(ctx, 18);
```

Comment:
nit: may be better to pass in value 13 instead of 18. 

Also the naming is a bit challenging for the last version, because it may apply to future versions also (like version 19). So, I wonder if it is better to just call them `resizeShapeInference_opset13` and `resizeShapeInference_opset11` ? The upper end is implicit and omitted from name.